packages/core/src/part.mjs

```diff
@@ -78,6 +78,17 @@ Part.prototype.attr = function (name, value, overwrite = false) {
   return this
 }
 
+/**
+ * Gets a free ID to use in the part
+ *
+ * @return {string} id - A free ID to use
+ */
+Part.prototype.getId = function (prefix = '') {
+  this.freeId += 1
+
+  return prefix + this.freeId
```

Comment:
I'm not liking that this is repeated code from
```
/**
 * Returns a closure holding a getId method (returns an ID unused in this part)
 *
 * @return {function} getId - The getId function
 */
Part.prototype.__getIdClosure = function () {
  const self = this
  const method = function (prefix = '') {
    self.freeId += 1

    return prefix + self.freeId
  }

  return method
}
```

One has to reference the other. Your choice which

Reply:
I've called the closure instead.